src/ClientData/include/ClientData/CaptureData.h

```diff
@@ -255,12 +257,17 @@ class CaptureData {
       uint64_t scope_id, uint64_t min_tick = std::numeric_limits<uint64_t>::min(),
       uint64_t max_tick = std::numeric_limits<uint64_t>::max()) const;
 
+  [[nodiscard]] const orbit_grpc_protos::CaptureStarted& GetCaptureStarted() const {
```

Comment:
You could consider calling this `GetCaptureStartedEvent` as it's a `CaptureEvent`, maybe it's a bit more specific. No strong opinion though.